epimodel/exports/epidemics_org.py

```diff
@@ -103,27 +114,32 @@ def extract_smallish_data(
         hopkins: Optional[pd.DataFrame],
         foretold: Optional[pd.DataFrame],
         timezones: Optional[pd.DataFrame],
+        un_age_dist: Optional[pd.DataFrame],
     ) -> Dict[str, Dict[str, Any]]:
-        d = {
-            "Rates": rates.replace({np.nan: None}).to_dict()
-            if rates is not None
-            else None,
-            "JohnsHopkins": {
+        d = {}
+
+        if rates is not None:
```

Comment:
This means that the key is not being present at all in the result dict. Is that fine on JS?

Reply:
Yes that's fine